app/design/frontend/base/default/layout/aligent_timedbanners.xml

```diff
@@ -0,0 +1,11 @@
+<layout>
+    <default>
+        <reference name="root">
```

Comment:
This outer reference should be redundant.  Have you tried the layout without it?